<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>btn_7Class</class>
 <widget class="QMainWindow" name="btn_7Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>543</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>btn_7</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/return/return.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>110</y>
      <width>521</width>
      <height>411</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文楷体</family>
      <pointsize>18</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文楷体</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>逾期名单</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>60</y>
      <width>75</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文楷体</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>显示</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
